atrackerv1/atracker_v1_examples_test.go

```diff
@@ -115,7 +113,7 @@ var _ = Describe(`AtrackerV1 Examples Tests`, func() {
 				panic(err)
 			}
 			b, _ := json.MarshalIndent(target, "", "  ")
-			fmt.Println(string(b))
+			fmt.Printf("\nCreateTarget() result:\n %s \n", string(b))
```

Comment:
Using space around the `%s` causes "not that pretty" indentation:
```
UploadFile() result:
 {
  "id": "xxx",
  "filename": "example.log",
  "size_in_bytes": 42,
  "created_at": "2021-03-23 16:41:50",
  "url": "xxx"
}
```
```suggestion
			fmt.Printf("\nCreateTarget() result:\n%s\n", string(b))
```